### PR TITLE
gather and store logs for scylla manager

### PR DESCRIPTION
**Pull request by aleksbykov**
- adding method for gather scylla manager log
- refactor gathering and storing db_nodes and monitoring nodes additional data

### CODE REVIEW

File: sdcm/cluster.py [hunk 1]
```diff
@@ -2262,6 +2270,22 @@ def restart_scylla(self, nodes=None):
             node.start_scylla(verify_up=True)
             self.log.debug("'{0.name}' restarted.".format(node))
 
+    def gather_additional_information(self, storing_dir):
```
Comment (bentsi) [inline sdcm/cluster.py]: `gather_additional_information` not much informative. Which information? Why to gather it? Why is it additional?  In code we collect/get/retrieve data or logs. When giving names to the method ask yourself, if somebody that is new to the code and sees only this method, can he understand what it does without reading method implementation?
Change to smth. like `gather_additional_information` ->  `collect_logs`

File: sdcm/db_stats.py [hunk 2]
```diff
@@ -334,7 +333,7 @@ def get_system_details(self):
                 'sys_info': node.get_system_info(),
             }
             system_details.update(node_system_info)
-        archive = self.archive_system_details_information(files_to_archive)
+        archive = self.archive_additional_test_details()
```
Comment (bentsi) [inline sdcm/db_stats.py]: `archive` -> `archive_path`

File: sdcm/db_stats.py [hunk 3]
```diff
@@ -436,44 +435,15 @@ def calculate_stats_average(self):
...
-        For each node prepare relevant files on node, copy them to local directory
-        <cluster_log_dir>/system_data , write to files the installed
-        packages on node, instace console output and screenshot on
-        aws deployment, arhcive contents of appropriate directory and upload to
-        S3 storage.
-
-        Arguments:
-            files_to_archive {list} -- list of files as fullpath which should be stored as
-            system details data
+    def archive_additional_test_details(self):
```
Comment (bentsi) [inline sdcm/db_stats.py]: same here: `archive_additional_test_details` -> `archive_logs`

File: sdcm/cluster.py (hunk 1 above)
Comment (aleksbykov) [reply sdcm/cluster.py]: renamed

File: sdcm/db_stats.py (hunk 2 above)
Comment (aleksbykov) [reply sdcm/db_stats.py]: renamed

File: sdcm/db_stats.py (hunk 3 above)
Comment (aleksbykov) [reply sdcm/db_stats.py]: renamed

File: sdcm/cluster.py [hunk 4]
```diff
@@ -3265,3 +3309,6 @@ def download_monitor_data(self):
 
     def destroy(self):
         pass
+
+    def gather_additional_information(self, storing_dir):
```
Comment (bentsi) [inline sdcm/cluster.py]: renamae here too

**Pull request merged**